tests/foreman/cli/test_discoveryrule.py

```diff
@@ -240,75 +283,37 @@ def test_positive_create_disabled_rule(self, discoveryrule_factory):
         rule = discoveryrule_factory(options={'enabled': 'false'})
         assert rule.enabled == 'false'
 
-    @pytest.mark.tier3
+    @pytest.mark.tier1
+    @pytest.mark.parametrize('host_name', **parametrized(invalid_hostnames_list()))
     @pytest.mark.parametrize('name', **parametrized(invalid_values_list()))
-    def test_negative_create_with_invalid_name(self, name, discoveryrule_factory):
-        """Create Discovery Rule with invalid names
+    def test_negative_create_rule_with_discovery_params(
+        self, name, host_name, discoveryrule_factory
+    ):
+        """Create Discovery Rule with negative parameters.
 
-        :id: a0350dc9-8f5b-4673-be88-a5e35d1f8ca7
+        :id: 0906cf64-ed0b-49af-844f-1af22f81ab94
 
         :expectedresults: Error should be raised and rule should not be created
 
         :CaseImportance: Medium
-
-        :CaseLevel: Component
-
-        :parametrized: yes
         """
         with pytest.raises(CLIFactoryError):
             discoveryrule_factory(options={'name': name})
 
-    @pytest.mark.tier3
-    @pytest.mark.parametrize('name', **parametrized(invalid_hostnames_list()))
-    def test_negative_create_with_invalid_hostname(self, name, discoveryrule_factory):
```

Comment:
I think it'll be better to keep this separate test for negative hostnames, instead of having new tests with parametrized cominations of invalid_hostnames_list and invalid_values_list

Reply:
I think the parametrization checks more combination so good to keep it that way